orms/gorm/user.go

```diff
@@ -1,58 +1,116 @@
 package gorm
 
+import "gorm.io/gorm"
+
 type User struct {
 	Id        string `json:"id"`
+	FirstName string `json:"firstName" gorm:"column:first_name"`
```

Comment:
is there a way to specify nullable fields in gorm?

Reply:
yes sure